Sources/Profile-View.php

```diff
@@ -1239,10 +1237,10 @@ function list_getUnwatched($start, $items_per_page, $sort, $memID)
 		SELECT lt.id_topic
 		FROM {db_prefix}log_topics as lt
 			LEFT JOIN {db_prefix}topics as t ON (lt.id_topic = t.id_topic)
-			LEFT JOIN {db_prefix}boards as b ON (t.id_board = b.id_board)
 			LEFT JOIN {db_prefix}messages as m ON (t.id_first_msg = m.id_msg)' . (in_array($sort, array('mem.real_name', 'mem.real_name DESC', 'mem.poster_time', 'mem.poster_time DESC')) ? '
 			LEFT JOIN {db_prefix}members as mem ON (m.id_member = mem.id_member)' : '') . '
-		WHERE lt.id_member = {int:current_member}
+		WHERE {query_see_message}
+			AND lt.id_member = {int:current_member}
 			AND unwatched = 1
 			AND {query_see_board}
```

Comment:
You added `{query_see_message}` at the start of this WHERE clause, but forgot to remove this line at the end of the clause.